### PR TITLE
Don't recreate exists ZkNode

### PR DESCRIPTION
**Pull request by tienlemanh**
writeState always try to create ZkNode even the ZkNode is created. That's much for zookeeper.

### CODE REVIEW

**Comment by avsej** [comment]
Could you submit the change through the http://review.couchbase.org? More info about contributing http://developer.couchbase.com/open-source-projects

File: src/main/java/com/couchbase/kafka/state/ZookeeperStateSerializer.java [hunk 1]
```diff
@@ -106,11 +111,23 @@ private String pathForState(final short partition) {
         return String.format("/couchbase-kafka-connector2/%s/%d", bucket, partition);
     }
 
+    private Boolean isCreatedPersistentZkNode(String path) {
```
Comment (avsej) [inline src/main/java/com/couchbase/kafka/state/ZookeeperStateSerializer.java:114]: why not unbox the boolean when returning it?

**Comment by avsej** [comment]
I opened ticket about general issue to verify behaviour [KAFKAC-44](https://issues.couchbase.com/browse/KAFKAC-44)

Are you going to revisit this patch and submit it through the gerrit?

**Comment by tienlemanh** [comment]
How can I commit over gerrit? I pushed the commit to http://review.couchbase.org/couchbase-kafka-connector/  but I got an error `Upload denied for project 'couchbase-kafka-connector'`

**Comment by avsej** [comment]
@brett19 could you take a look at this?

**Comment by brett19** [comment]
@avsej What would you like me to do in this case?

**Comment by avsej** [comment]
Thank you I will re-implement it in next release. Please track the aforementioned ticket.

**Pull request closed without merging**